src/entities/dataset/lookup-table.ts

```diff
@@ -26,6 +26,9 @@ export class LookupTable extends BaseEntity implements FileImportInterface {
   @Column({ type: 'varchar', length: 255 })
   filename: string;
 
+  @Column({ name: 'original_filename', type: 'varchar', length: 255 })
```

Comment:
just use `type: text` (it's basically identical in Postgres and you don't need to worry about length)

Reply:
Yeah but here I want to worry about length... Windows still has a 255 character limit on filenames and while it's unlikely someone might violate that it's still a possibility.  Also I might upload a file from my Mac which is then attempted to be downloaded to a Windows machine.